scripts/workers/submission_worker.py

```diff
@@ -641,18 +656,29 @@ def main():
     if challenge_pk:
         q_params["pk"] = challenge_pk
 
-    challenges = Challenge.objects.filter(**q_params)
-    for challenge in challenges:
-        load_challenge(challenge)
+    if settings.DEBUG or settings.TEST:
+        if settings.LIMIT_CONCURRENT_SUBMISSION_PROCESSING:
+            if not challenge_pk:
+                logger.exception(
+                    "Please add CHALLENGE_PK as environment variable"
```

Comment:
Please be more explicit in the error message. If I am a new developer, then I would not know what is going on even after seeing this error.